elephant/test/test_spade.py

```diff
@@ -698,25 +701,25 @@ def test_signature_significance_fdr_bh_corr(self):
             alpha=0.15, winlen=1, corr='fdr_bh')
         self.assertEqual(sig_spectrum, [(2., 3., False), (2., 4., True)])
 
-    def test_different_surrogate_method(self):
-        np.random.seed(0)
-        random.seed(0)
-        spiketrains = [stg.homogeneous_poisson_process(rate=20*pq.Hz)
-                       for _ in range(2)]
-        surr_methods = ('dither_spikes', 'joint_isi_dithering',
-                        'bin_shuffling',
-                        'dither_spikes_with_refractory_period')
-        pv_specs = {'dither_spikes': [[2, 2, 0.8], [2, 3, 0.2]],
-                    'joint_isi_dithering': [[2, 2, 0.8]],
-                    'bin_shuffling': [[2, 2, 1.0], [2, 3, 0.2]],
-                    'dither_spikes_with_refractory_period':
-                        [[2, 2, 0.8]]}
-        for surr_method in surr_methods:
-            pv_spec = spade.pvalue_spectrum(
-                spiketrains, bin_size=self.bin_size,
-                winlen=self.winlen, dither=15*pq.ms,
-                n_surr=5, surr_method=surr_method)
-            self.assertEqual(pv_spec, pv_specs[surr_method])
+    # def test_different_surrogate_method(self):
+    #     np.random.seed(0)
+    #     random.seed(0)
+    #     spiketrains = [stg.homogeneous_poisson_process(rate=20*pq.Hz)
+    #                    for _ in range(2)]
+    #     surr_methods = ('dither_spikes', 'joint_isi_dithering',
+    #                     'bin_shuffling',
+    #                     'dither_spikes_with_refractory_period')
+    #     pv_specs = {'dither_spikes': [[2, 2, 0.8], [2, 3, 0.2]],
+    #                 'joint_isi_dithering': [[2, 2, 0.8]],
+    #                 'bin_shuffling': [[2, 2, 1.0], [2, 3, 0.2]],
+    #                 'dither_spikes_with_refractory_period':
+    #                     [[2, 2, 0.8]]}
+    #     for surr_method in surr_methods:
+    #         pv_spec = spade.pvalue_spectrum(
+    #             spiketrains, bin_size=self.bin_size,
+    #             winlen=self.winlen, dither=15*pq.ms,
+    #             n_surr=5, surr_method=surr_method)
+    #         self.assertEqual(pv_spec, pv_specs[surr_method])
```

Comment:
What's up with this test?

Reply:
Not sure -- it does pass for me, also appear to be functional. @stellalessandra Do you know why this one is commented out?

Reply:
Ok, I see the point, I'd agree this would be somewhat redundant, and may break in case of changes to the implementations of the surrogate methods, even if those are correct (i.e.: maybe after an update of the algorithm, a surrogate method produces a different realization with a given random seed -- in that case this test would start to fail, but there would be no bug.